pymc/tests/test_bart.py

```diff
@@ -59,7 +58,24 @@ def test_missing_data():
         mu = pm.BART("mu", X, Y, m=10)
         sigma = pm.HalfNormal("sigma", 1)
         y = pm.Normal("y", mu, sigma, observed=Y)
-        idata = pm.sample(random_seed=3415)
+        idata = pm.sample(random_seed=3415, chains=1)
+
+
+@pytest.mark.xfail(reason="random fn is not yet implemented")
```

Comment:
Why does this PR affect BART at all?

Reply:
This PR met times when everything was fragile and not implemented and had a broken misconfigured test suite. When I configured test suite properly Bart failed

Reply:
I should go revisite these places once again

Reply:
If it is the case that there was an unrelated issue with BART, can you put those in a separate PR or, alternatively, clean the PR commits into VI and non VI commits that won't be squashed when merged?

Reply:
I removed these changes from the file. Seems it is now being tested in some way